### PR TITLE
Welcome to the Virtual Open Space!

### PR DESCRIPTION
**Pull request by brunobord**
Summary: How to cope with remote worker isolation using a virtual open-space.

### CODE REVIEW

File: content/welcome-to-the-virtual-open-space.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* In this virtual space, it's **impossible to have more than one conversation at a time**. If you need to have a specific conversation that takes more than 5 mins or so and could distract / annoy others, create a dedicated chatroom for the people concerned, and shift to this one. Of course, you can always come back once you're done!
+
+### A few years in...
+
+Sometimes this place is crowdy. Sometimes, nobody's here, for various reasons. Maybe everybody's on vacation. Or they all are trapped in a meeting. Maybe they're not in the right mood to enter this space.
+
+Sometimes you think you always see the same faces (or avatars). But from time to time, a new colleague joins in, and they bring something new and refreshing.
+
+But I think we've achieved ou main objective, that is to say bring back a *place to socialize* with people you work with, even if they're a thousand kilometers away. You won't feel isolated again, because at any time of the working day, you can find a friendly face, have a chat around, share your feelings, tell jokes or ask for advices... if not "in person", at least it's much better than a text-only conversation via email or Slack.
```
Comment (aRkadeFR) [inline content/welcome-to-the-virtual-open-space.md]: our* main objective

Comment (Dsaad-People-Doc) [reply content/welcome-to-the-virtual-open-space.md]: * "place to socialize with your coworkers" avoids the awkward "with people you work with"

* "chat" instead of "have a chat around"

* "ask for advice" instead of "ask for advices"

Overall: pretty good.

**Comment by aRkadeFR** [comment]
The subject is something important for me and that's the reason I will leave my complete feedback / review of the PR. My mother tongue is not English, so please bare with my English.
I won't be offended if you don't read it or don't take into account.
TL;DR: I mostly agree with the fundamental idea and the under lying ideas of the article, I like the way you're phrasing your English @brunobord , but I think the organization of the paragraphs/structure of the article is a big negative side of the article.

I will spend some time trying to explain my negative feeling when I read the article, and how that could be slightly changed to make it more persuasive and positive.

The article starts by:
> Eventually, some of us may feel isolated.

I prefer keeping some positive goal in mind instead of trying to fix a negative feeling. 
That could be rephrased for something like: "As a great remote spirit in the R&D at PeopleDoc has been present from the beginning, we want to keep this positive spirit alongside the great growth of the company. Doing so requires some efforts from everyone and some simple rules."
This is important from my point of view, as it stated "some of us may feel isolated", but doesn't explain neither why or where this isolation comes from.

> It was open all day long during office hours, and it helped a lot solving issues that couldn't have been solved using text chat or email.

Same as my last point. Why the text chat or email couldn't solve the issues? That feels like a strong critic towards email and text chat, and as open-source world use heavily emails (mailing list) and text chat (IRC, slack and so on), the reader, as I read this part, assumes PeopleDoc is not using correctly email / text chat (which is not entirely false). But in a general way, I prefer to put the perspective more positive without the critics. For instance: "Setting up a video conference during this hackathon felt beneficial for all the participant for couple of reason. First of all we didn't spent any time synchronizing as video conference is a real time communication channel. And second of all, we felt closer to each other, because of the tone of the voice, the rhythm of the communication and so on."

Last point about the article is the subject about "creating new channel when a subject comes up for not bothering the colleagues present in the open-space video conference". This part lacks of facts and rules to make it clear how to handle this problem of becoming noisy in the video conference. I would maybe remove this part from the article, for having an article that can apply every case/company.

Thank you for the time you took to write this article.

**Comment by jpic** [comment]
Great article with quiet some food for thought, even freelancers and employees across companies have visioconference channels or phoneconference sometimes started to pair code on something but then keeps going on even when people switch projects, get help rather than be stuck results in a better experience at work from the employee perspective and better ROI from a financial perspective.

Unlike @aRkadeFR, I like how this article looks like a real experience rather than a an experience travestite into marketing bs, or even toneless "objective" articles blatantly lacking any taste at all. I mean that the article is not talking about a machine subject to standard rules and where people have to follow a procedure (or make one for any new case) as taylorism would see it, but rather primarly as a Human system where people are trusted and empowered to adapt their behaviour to the environment using their inteligence.

Thanks for the completely unexpected credit, 

with LOVE <3

**Comment by brunobord** [comment]
@aRkadeFR 

> I won't be offended if you don't read it or don't take into account.

fair enough. On my side, I didn't feel aggressed by your comments, and I understand most of your points.

> The article starts by:
> 
>    > Eventually, some of us may feel isolated.
> 
> I prefer keeping some positive goal in mind instead of trying to fix a negative feeling.
> That could be rephrased for something like: "As a great remote spirit in the R&D at PeopleDoc has been present from the beginning, we want to keep this positive spirit alongside the great growth of the company. Doing so requires some efforts from everyone and some simple rules."
> This is important from my point of view, as it stated "some of us may feel isolated", but doesn't explain neither why or where this isolation comes from.

I tought it was self-explanatory. I'm not in people's mind, and the sense of isolation is a common feeling among remote workers ; not all, of course, and not all the time. I can eventually rephrase this to let it less ambiguous about this cause.

I understand your point, about "keep some positive goals" though, but I can't see I could rewrite this without altering the underlying structure. It's pretty much like a bug report: you've got the current situation/context, then the symptoms, then the root cause, then the implementation of a solution. Maybe to be worded better like this: the virtual open-space was more a response to a **frustration**, rather than a real **problem**.

> 
>    > It was open all day long during office hours, and it helped a lot solving issues that couldn't have been solved using text chat or email.
>
> Same as my last point. Why the text chat or email couldn't solve the issues?

All right, maybe it was a bit terse. I tend to think, and again, this is something shared by many others, that a face-to-face conversation has a better impact on any discussion. Yes, we can use text-only for a lot of things, express opinions, etc. But you've got to admit that voice has a better bandwidth. There are so many ways to understand a written statement without feeling the irony, or the joke, etc.


> That feels like a strong critic towards email and text chat [..]

It is not. See? you're misinterpreting my words... ;o)

It's not that this "text-only" medium is *bad* or *good*. No medium is *good* or *bad*. It depends on the *context* (and by context, I mean the situation and the people involved. some are gifted to express their point using text (and English), some aren't).

For the hackathon, the situation was hard to handle because I felt unable to explain my problems using text-only, and we needed to pair-prog ; sharing screen and keyboards ; show outputs, scroll up and down to find the cause of the bugs, rerun builds and inspect the output "on the fly", etc. It would have been "doable" using only email or IRC... but it would have taken countless messages, back and forth and a much longer period of time.

> Last point about the article is the subject about "creating new channel when a subject comes up for not bothering the colleagues present in the open-space video conference". This part lacks of facts and rules to make it clear how to handle this problem of becoming noisy in the video conference. I would maybe remove this part from the article, for having an article that can apply every case/company.
> 

I guess that it's called "common sense". If the conversation is a bit too specific, do you think you are disturbing? If all the persons present are not directly concerned, do you disturb them? Is the "off-topic" conversation is too long, is it mandatory that the people would leave? How long is "too long?"
Maybe some persons would be capable to "filter out" smalltalk conversations, while some others can't focus... Not everybody has the same limits and I don't think that you can enforce it with established rules.

> Thank you for the time you took to write this article.

And thank you for your comment.

**Comment by brunobord** [comment]
I've pushed a few fixups, including some suggestions by @aRkadeFR ; if it's okay, I'll rebase+merge.
